### PR TITLE
Improved code to make search and filters section sticky on all find pages(#24b11cp)

### PR DESCRIPTION
**Pull request by azkyakhan**
https://user-images.githubusercontent.com/82817616/160068690-5dba91ad-3310-4d37-8b29-1bf9b16fdeee.mp4

### CODE REVIEW

File: src/theme/variables.css [hunk 1]
```diff
@@ -277,6 +277,9 @@ body {
 
 .filters {
   grid-area: filters;
+  position: sticky;
```
Comment (dt2patel) [inline src/theme/variables.css:280]: I think this should be desktop only right?

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/theme/variables.css (hunk 1 above)
Comment (azkyakhan) [reply src/theme/variables.css:280]: Filters are already visible desktop-only which is why I have not written this styling on desktop-only media query.

Comment (dt2patel) [reply src/theme/variables.css:280]: The same component will be used in the mobile view so the base class has to work for both devices.

**Comment by dt2patel** [comment]
Will pick this later since right now we don't know how this will fit into our UX

**Pull request closed without merging**